core/src/main/java/org/mskcc/cbio/oncokb/model/OncoKBInfo.java

```diff
@@ -49,6 +52,12 @@ public OncoKBInfo() {
         } else {
             this.publicInstance = false;
         }
+
+        try {
+            this.genomeNexus = GenomeNexusUtils.getParsedGenomeNexusVersion();
+        } catch (RestClientException e) {
+            this.genomeNexus = null;
```

Comment:
@calvinlu3 do you think it'll be confusing to set genomeNexus to null on a fail API request? I feel like throwing a 5xx error isn't really what we want so this should be fine.